client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -152,6 +152,23 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
         }
     };
 
+    public getTitleWithChanges = (motion: ViewMotion, change: ViewUnifiedChange, crMode: ChangeRecoMode): string => {
```

Comment:
please use the `ChangeRecommendationRepositoryService` class or create a new service